airbyte-webapp/src/views/Connection/ConnectionForm/formConfig.tsx

```diff
@@ -28,8 +28,9 @@ import { ValuesProps } from "hooks/services/useConnectionHook";
 import { useCurrentWorkspace } from "services/workspaces/WorkspacesService";
 
 import calculateInitialCatalog from "./calculateInitialCatalog";
+import { ConnectionOrPartialConnection } from "./ConnectionForm";
 
-interface FormikConnectionFormValues {
+export interface FormikConnectionFormValues {
```

Comment:
It looks like this is just slightly different from the `ValuesProps` type.  Would it make sense to have them reference each other somehow?

Reply:
I'm not sure it's worth trying to bridge the types. There's differences in what's optional.